sorc/grid_tools.fd/regional_esg_grid.fd/pmat5.f90

```diff
@@ -154,24 +152,23 @@ subroutine dinivmap(alon0,alat0,rot3)!                               [inivmap]
 rot3(3,3)=slat0
 end subroutine dinivmap
 
-!=============================================================================
+!> Transform "Cartesian" to "Geographical" coordinates, where the
+!!   geographical coordinates refer to latitude and longitude (radians)
+!!   and cartesian coordinates are standard earth-centered cartesian
+!!   coordinates: xe(3) pointing north, xe(1) pointing to the 0-meridian.
+!!  XE       three cartesian components.
```

Comment:
What formatting are we doing for these three lines?

Pehaps you want a list:
```
- list item 1 
- list item 2
- list item 3

```

Reply:
???

Reply:
xe, rlat, and rlon are belong to parameter. Not a list.